biome.json

```diff
@@ -106,7 +106,7 @@
         "formatter": {
             "jsxQuoteStyle": "double",
             "quoteProperties": "asNeeded",
-            "trailingCommas": "none",
+            "trailingComma": "none",
```

Comment:
trailingComma is deprecated in biome 1.8.3, which is what the project now uses
use `bun install @biomejs/biome@1.8.3` if you haven't upgraded it yet, even though it's in package.json

Reply:
Oh, sorry, I should've updated biome.